### DISCUSSION TITLE
Find a way to display Token values on a layer in Developer handoff

### DISCUSSION THREAD

**Discussion by six7** [six7/figma-tokens#148] [Ideas]
We should allow Developer who don't have a Figma account to see the Token values that are on a layer in order for the developer to be able to inspect what tokens were used on a given layer

**Comment by phantasmagoriadigital** [comment]
Do you have any plans to take this feature into production any time in the near future? Documenting a design system right now and this is exactly what I was trying to find a solution for :-)

**Comment by six7** [comment]
I was hoping for a solution from Figma, where pluginData could be somehow shown in the Inspect tab for Viewers to see.

Documentation tokens provide a good way to do these things, a lot of teams simply insert some sort of token sheet beneath their components to illustrate what was used https://docs.tokens.studio/tokens/documentation-tokens

However I'd like to go in the direction of Redlines, where you maybe could duplicate your component, hit the "Redline tokens" button and the plugin would do kind of what the Redlines plugin does: Insert token values and names in Redline manner. This would require some sort of effort to document your components, but I think it's worth it.

**Reply by phantasmagoriadigital** [reply]
@six7 Are you on the design systems slack? Somebody there built a small poc that does a tokens documentation visually. Some other cool stuff going on as well.

**Reply by six7** [reply]
Ooh no I'm not, can you send me an invite?

**Comment by phantasmagoriadigital** [comment]
I think the redlines way is the way to go, handoff in Figma isn't always a solution. One scenario is when all handoff happens through Zeplin. Also, for creating documentation this doesn't serve the purpose.
